packages/odyssey-react/src/components/Radio/RadioGroup/index.tsx

```diff
@@ -87,7 +87,7 @@ const RadioGroup = forwardRef<HTMLFieldSetElement, Props>((props, ref) => {
 
   const legendElement = (
     <legend
-      className="ods-input-legend"
+      className="ods-field--label"
```

Comment:
label / legend ?

Having the class refer to the element and yet have them out of sync feels counter-intuitive. Can we fix this?

Reply:
This is intentional. The class here is referring to the semantic purpose of the component `Field.Label`, not the element itself. Since `Label` is the customary language for referring to this piece of UI, it seems more author-friendly. Devs/designers should not need to worry about the element being used (that should happen automatically). 

If that isn't satisfactory, we can rename this component to `Field.Title`.